file_writer_control/JobHandler.py

```diff
@@ -76,6 +76,12 @@ def set_stop_time(self, stop_time: datetime) -> Union[CommandHandler, None]:
             current_status.service_id, self._job_id, stop_time
         )
 
+    def stop_now(self) -> Union[CommandHandler, None]:
```

Comment:
If I understood correctly, `stop_now` will eventually be sending `stop_time = 0` (as before) rather than `datetime.now()` in the Stop message to FW. During sprint I think it was changed to `datetime.now()`.  Just to confirm.

Reply:
Ok, you are probably right about that. I can't be bothered to check. :-) As mentioned in the PR-description, I brought back this function as its loss caused crashes in applications that were using it.